install-dev/data/xml/feature_flag.xml

```diff
@@ -12,5 +12,6 @@
   <entities>
     <feature_flag id="product_page_v2" name="product_page_v2" label_wording="New product page - Single store" label_domain="Admin.Advparameters.Feature" description_wording="This page benefits from increased performance and includes new features such as a new combination management system." description_domain="Admin.Advparameters.Help" state="0" stability="beta" />
     <feature_flag id="product_page_v2_multi_shop" name="product_page_v2_multi_shop" label_wording="New product page - Multi store" label_domain="Admin.Advparameters.Feature" description_wording="Access the new product page, even in a multistore context. This is a work in progress and some features are not available." description_domain="Admin.Advparameters.Help" state="0" stability="beta" />
+    <feature_flag id="attribute_group" name="attribute_group" label_wording="Attribute group" label_domain="Admin.Advparameters.Feature" description_wording="Enable / Disable migrated attribute group page. This is a work in progress and some features are not available." description_domain="Admin.Advparameters.Help" state="0" stability="beta" />
```

Comment:
```suggestion
    <feature_flag id="attribute_group" name="attribute_group" label_wording="Attribute group" label_domain="Admin.Advparameters.Feature" description_wording="Enable / Disable migrated attribute group page." description_domain="Admin.Advparameters.Help" state="0" stability="beta" />
```

Reply:
I think we can keep a generic wording here, or we'll have to copy this on every feature flag. The fact that it's still in `beta` stability should be enough, as we can add a common warning message for all those feature flags on the page. This way once it's stable we just need to update its stability field and no need to update the description.

Reply:
Same modif should be applied in the `FeatureLang` file